plugins/search-backend-node/src/Scheduler.ts

```diff
@@ -56,11 +57,7 @@ export class Scheduler {
     this.schedule.forEach(({ task, interval }) => {
       // Fire the task immediately, then schedule it.
       task();
```

Comment:
Hm so ... this one isn't awaited. And then an overlapping one will be started immediately, right? Feels like we could skip this line as long as `runPeriodically` does fire right away

Reply:
Ohhh, you are so correct! Missed to delete that line when taking it over from my fork...